### DISCUSSION TITLE
Recommendations for storage

### DISCUSSION THREAD

**Discussion by rustygreen** [cloudnative-pg/cloudnative-pg#1005] [General]
Do you recommend using block storage (such as longhorn or rook/ceph) or local storage? I know this is a loaded question, but I'm looking for guidance on whether block storage would really offer any benefit if backups are being created for the database (hence block storage replication is not really necessary)? I'd prefer to keep things simple and keep block storage out of the mix, but just wanted to make sure I wasn't missing some unkown benefit of using block storage with cloudnative-pg.

I guess my main question is, are there any drawbacks or considerations to keep in mind of using block storage with cnpg (such as longhorn)?

I have a conversation on this topic, with some background information here: https://www.reddit.com/r/kubernetes/comments/ym0ol5/block_storage_for_postgres_cluster/

Thanks, for any guidance!

**Comment by guettli** [comment]
I am investigating this, too.

After reading the docs, and a reply from the creators of cnPG on slack, I will use local node storage.


https://cloudnative-pg.io/documentation/1.18/controller/

https://cloudnative-pg.io/documentation/1.18/storage/

https://app.slack.com/client/T03BCEDRZL1/C03AT6KSKDL
